scripts/ci/gitlab/pipeline/test.yml

```diff
@@ -42,6 +42,9 @@ test-linux-stable:
     RUSTFLAGS: "-Cdebug-assertions=y -Dwarnings"
   script:
     - time cargo test --workspace --profile testnet --verbose --locked --features=runtime-benchmarks,runtime-metrics,try-runtime,ci-only-tests
+    # Run `polkadot-runtime-parachains` tests a second time because `paras_inherent::enter` tests are gated by not having
+    # the `runtime-benchmarks` feature enabled.
+    - time cargo test --workspace --profile testnet --verbose --locked --features=runtime-metrics,try-runtime,ci-only-tests -p -p polkadot-runtime-parachains 
```

Comment:
Should I this as another step in the pipeline to parallelize execution ?

Reply:
> Should I add this as another step in the pipeline to parallelize execution ?

What's the expected runtime of the new test?

Reply:
Hmm, looks like it's almost 20min. Could we narrow down the second test run further to just the tests that are missed by the first? Otherwise a new job would make sense imo.

Reply:
this line was all wrong, re-ran all the tests actually.

Reply:
real	1m14.137s